lib/Doctrine/DBAL/Platforms/OraclePlatform.php

```diff
@@ -922,6 +928,37 @@ public function getColumnDeclarationSQL($name, array $field)
         return $name . ' ' . $columnDef;
     }
 
+    /**
+     * This function is a copy of getColumnDeclarationSQL.
+     * The only difference is that the data type is not in the column declaration.
+     * This declaration is particularly useful when some property of the column changes
+     *  (e.g. nullable, default value), but the data type does not change.
+     */
+    public function getColumnDeclarationWithoutTypeSQL($name, array $field)
```

Comment:
This method doesn't need to be public.
```suggestion
    private function getColumnDeclarationWithoutTypeSQL($name, array $field)
```